apps/test-tsx-app/utils/prici-runner.ts

```diff
@@ -6,8 +6,7 @@ import { initialize } from '@prici/sdk';
 const token = 'abcd';
 const tenantName = 'demo-tenant-' + Math.random();
 
-
-async function runPriciInstance() {
+async function runPriciInstance(isAdminUiEnabled: string) {
```

Comment:
prefer to retrieve options as an object:

```suggestion
async function runPriciInstance({ adminUi }: { adminUi?: boolean } = {}) {
```